cime_config/config_component.xml

```diff
@@ -303,11 +304,12 @@
 
   <entry id="CAM_LINKED_LIBS">
     <type>char</type>
-    <valid_values></valid_values>
+    <valid_values>-lmpas</valid_values>
```

Comment:
I might vote to not worry about updating `valid_values` here, as I could imagine all sorts of different libraries that might be linked in the future, and we probably don't want to require future users/developers to have to update this line.  

Plus if someone mistakenly tries to link the wrong library it will probably either be ignored or cause an obvious compiler or linker error at build time.

Reply:
Of course. I reverted this change in https://github.com/ESCOMP/CAM-SIMA/pull/245/commits/3a9b8ad2c0ceecf740a60c40dfade1c78eff1f74.